verify/BUILD

```diff
@@ -10,7 +10,6 @@ sh_test(
     name = "verify-pylint",
     srcs = ["verify-pylint.sh"],
     data = [
-        ":pylint_bin",
```

Comment:
Can the `pylint_bin` py_binary target be removed entirely now?

Reply:
This won't work as-is, `pylint_bin` is being used as a hack to set up the deps we import :(
It worked locally because I have all of those things installed, but it's definitely not hermetic and it doesn't work right in CI. I'm not sure if we need pylint to be hermetic though...